### PR TITLE
I added more log and Complete rewrite for Portuguese language

### PR DESCRIPTION
**Pull request by renatodolce**
I put to show log "loading_data" at the time it sticks and log "action_loading" when an action is requested.
image

### CODE REVIEW

File: scripts/filemanager.js [hunk 1]
```diff
@@ -1999,703 +1999,703 @@ $.richFilemanagerPlugin = function(element, pluginOptions)
...
+				if (!clipboard_model.hasCapability('paste') || clipboard_model.isEmpty()) {
+					return;
+				}
+				if (cbMode === null || cbObjects.length === 0) {
+					fm.warning(lg('clipboard_empty'));
+					return;
+				}
+
+				var targetPath = model.currentPath();
+				fm.write(lg('loading_data'));
```
Comment (renatodolce) [inline scripts/filemanager.js]: I put to show log "loading_data" at the time it sticks

File: scripts/filemanager.js [hunk 2]
```diff
@@ -3321,6 +3340,7 @@ $.richFilemanagerPlugin = function(element, pluginOptions)
 
 		$.each(items, function(i, item) {
 			deferred = deferred.then(function() {
+				fm.write(lg('action_loading').replace('%s', (successCounter+1)).replace('%s', totalCounter));
```
Comment (renatodolce) [inline scripts/filemanager.js]: I put to show log "action_loading" when an action is requested.

**Comment by renatodolce** [comment]
Sorry for any changes, but I just used to realign the code

File: languages/pt.json [hunk 3]
```diff
@@ -2,150 +2,151 @@
...
-    "action_move": "Move",
+    "action_download": "Baixar",
+    "action_extract": "Extrair",
+    "action_move": "Mover",
     "action_rename": "Renomear",
-    "action_replace": "Sobrepor arquivo",
+    "action_replace": "Substituir",
     "action_select": "Selecionar",
     "action_upload": "Enviar",
+    "action_loading": "Solicitação %s de %s em andamento.",
```
Comment (renatodolce) [inline languages/pt.json]: I added "action_loading" when an action is requested

Comment (renatodolce) [reply languages/pt.json]: image

File: languages/pt.json [hunk 4]
```diff
@@ -2,150 +2,152 @@
...
-    "upload_partially": "Some files are failed during upload.",
-    "version": "versão",
+    "summary_files": "Arquivos",
+    "summary_folders": "Pastas",
+    "summary_size": "Tamanho",
+    "summary_title": "Resumo de armazenamento",
+    "support_fm": "Filemanager é open source, por favor, nos ajudem!",
+    "type": "Tipo",
+    "unit_bytes": "Bytes",
+    "unit_tb": "Tb",
```
Comment (renatodolce) [inline languages/pt.json]: I added one more unit size "unit_tb"

File: scripts/filemanager.js [hunk 5]
```diff
@@ -2903,14 +2903,14 @@ $.richFilemanagerPlugin = function(element, pluginOptions)
...
-	// Converts bytes to KB, MB, or GB as needed for display
+	// Converts bytes to KB, MB, GB, or TB as needed for display
 	var formatBytes = function(bytes, round) {
 		if(!bytes) return '';
 		round = round || false;
 		var n = parseFloat(bytes);
 		var d = parseFloat(round ? 1000 : 1024);
 		var c = 0;
-		var u = [lg('unit_bytes'), lg('unit_kb'), lg('unit_mb'), lg('unit_gb')];
+		var u = [lg('unit_bytes'), lg('unit_kb'), lg('unit_mb'), lg('unit_gb'), lg('unit_tb')];
```
Comment (renatodolce) [inline scripts/filemanager.js]: Added new unit size for language: unit_tb

File: scripts/filemanager.js [hunk 6]
```diff
@@ -2903,14 +2903,14 @@ $.richFilemanagerPlugin = function(element, pluginOptions)
 		});
 	};
 
-	// Converts bytes to KB, MB, or GB as needed for display
+	// Converts bytes to KB, MB, GB, or TB as needed for display
```
Comment (renatodolce) [inline scripts/filemanager.js]: Update comment with TB size

**Comment by servocoder** [comment]
I can't accept your PR because of some incompatible changes, but I have updated pt.json file according to your commit. I also will think about implementation of upload progress bar.

**Pull request closed without merging**